test/adder/output/adder.mxsc.json

```diff
@@ -0,0 +1,66 @@
+{
+    "buildInfo": {
```

Comment:
Also delete the no longer used .wasm files. I pointed in the mx-sdk-rs PR how to easily automate this.